enrollment-server-onboarding/src/main/resources/application.properties

```diff
@@ -90,6 +90,7 @@ enrollment-server-onboarding.onboarding-process.max-error-score=15
 #enrollment-server-onboarding.onboarding-adapter.http-basic-auth-enabled=false
 #enrollment-server-onboarding.onboarding-adapter.http-basic-auth-username=
 #enrollment-server-onboarding.onboarding-adapter.http-basic-auth-password=
+#enrollment-server-onboarding.onboarding-adapter.headers.X-Custom-Header=foo
```

Comment:
This is a free-style way how to add any static header via `enrollment-server-onboarding.onboarding-adapter.headers.*`

Reply:
How exactly does this work?

Reply:
It is standard Spring binding functionality. `enrollment-server-onboarding.onboarding-adapter.headers` refers to `map` at https://github.com/wultra/enrollment-server/pull/369/files#diff-6f3dc223623821dca3239903563cad0c28bd56030a450f23fb611e6bba680bccR53 and `X-Custom-Header=foo` is an example of key `X-Custom-Header` and value `foo`.